components/config-provider/index.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { IconProvider } from '@ant-design/icons/lib/'; // keep last '/' since antd-tools need this
```

Comment:
Importing IconProvider like that is causing a bundle size problem due to all icons being imported on my side.

See #29038 (@zombieJ)

Reply:
webpack tree shaking can handle this. But yes, let optimize this import to `IconProvider` directly may save your problem.